### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by petermwash**

### CODE REVIEW

**Comment by petermwash** [comment]
Just made some more changes to my feature branch

File: UI/checkout.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			</div>
+		</div>
+	</main>
+	<footer class="page-footer">
+		<div>
+			<p>&copy; Created  with love by <a href="https://ke.linkedin.com/in/peter-mwaura-87b423107" target="_blank">pemwa</a></p>
+		</div>
+	</footer>
+</body>
+</html>
```
Comment (iankigen) [inline UI/checkout.html:89]: you should always have one new line at the end of a file

File: UI/myorderhstry.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+			</div>
+		</div>
+	</main>
+	<footer class="page-footer">
+		<div>
+			<p>&copy; Created  with love by <a href="https://ke.linkedin.com/in/peter-mwaura-87b423107" target="_blank">pemwa</a></p>
+		</div>
+	</footer>
+</body>
+</html>
```
Comment (iankigen) [inline UI/myorderhstry.html:82]: new line at the end of file

**Comment by iankigen** [comment]
Looks good to me. check new line at the end of file in your next commit.

**Comment by petermwash** [comment]
Thanks, I will keep that in mind in my next commit

**Pull request merged**